server/src/main/java/org/opensearch/http/DefaultRestChannel.java

```diff
@@ -147,6 +153,14 @@ public void sendResponse(RestResponse restResponse) {
             addCustomHeaders(httpResponse, restResponse.getHeaders());
             addCustomHeaders(httpResponse, threadContext.getResponseHeaders());
 
+            // Add server header
+            serverVersion.put(
+                SERVER_VERSION,
+                Arrays.asList(OPEN_SEARCH_NAME + "/" + Build.CURRENT.getQualifiedVersion() + " (" + Build.CURRENT.getDistribution() + ")")
```

Comment:
The `OPEN_SEARCH_NAME + "/" + Build.CURRENT.getQualifiedVersion() + " (" + Build.CURRENT.getDistribution() + ")"` could be extracted as a conststant:

```
static final String OS_VERSION = "OpenSearch/" + Build.CURRENT.getQualifiedVersion() + " (" + Build.CURRENT.getDistribution() + ")";
``

Reply:
Extracted